src/central-server/admin-service/core/build.gradle

```diff
@@ -38,6 +38,8 @@ dependencies {
     testImplementation('org.springframework.security:spring-security-test')
     testImplementation("org.hsqldb:hsqldb:$hsqldbVersion")
     testImplementation('org.liquibase:liquibase-core')
+    testImplementation 'org.xmlunit:xmlunit-core:2.9.0'
+    testImplementation 'org.xmlunit:xmlunit-assertj3:2.9.0'
```

Comment:
We should keep the formatting consistent - all other dependencies declared using brackets.